packages/adp-tooling/test/unit/preview/change-handler.test.ts

```diff
@@ -689,4 +689,108 @@ id=\\"btn-30303030\\""
             expect(generateChangeSpy).toHaveBeenCalled();
         });
     });
+
+    describe('addXmlFragmentForV4', () => {
```

Comment:
These tests should be in `descriptor-change-handler.test` file.

Reply:
renamed